addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -200,18 +199,20 @@ patch(Chatter.prototype, {
                 const text = partner.email ? partner.emailWithoutDomain : partner.name;
                 return `<span class="text-muted" title="${escape(
                     partner.email || _t("no email address")
-                )}">${escape(text)}</span>`;
+                )}"> ${escape(text)}</span>`;
             });
+        return markup(recipients);
+    },
+
+    get recipientsCount() {
```

Comment:
The name of this method makes it look like it returns a count/number, but it returns a text/string.